### PR TITLE
Countdown

### PR DESCRIPTION
**Pull request by kiasaki**
Following what was shared in slack:

screen shot 2016-12-13 at 4 58 02 pm

I made us a countdown page to put up for the 15th while we get a real website built.

cscreenshot
default
cscreenshot2
smaller screen
cscreenshot-error
mailchimp error

We'll just to have to double check MailChimp works once live (because locally you'll get a CORS error)

Open to feedback on the styles / colors I really just improvised with the GopherAcademy color scheme.

### CODE REVIEW

File: actions/app.go [hunk 1]
```diff
@@ -5,6 +5,7 @@ import (
 
 	"github.com/gopheracademy/gcon/actions/admin"
 	"github.com/gopheracademy/gcon/models"
+	"github.com/kiasaki/gcon/actions/countdown"
```
Comment (markbates) [inline actions/app.go:8]: This looks like it's accidentally using your fork, not the gopheracademy account.

File: actions/countdown/countdown.go [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package countdown
```
Comment (markbates) [inline actions/countdown/countdown.go:1]: Why is this ina different package from actions?

**Review by markbates — CHANGES_REQUESTED** [review]
while I'm not opposed to putting the countdown assets in a separate folder, I see no need to create a new package under `actions` for it.

**Comment by bketelsen** [comment]
While i appreciate the hand-crafted HTML/css/js, we bought the template because 95% of the people volunteering on this project have little html experience.  I'd prefer to stick to the template so we can all help on the frontend too.

**Comment by markbates** [comment]
Closing this based on @bketelsen feedback.

**Comment by kiasaki** [comment]
The separate package is really because having a separate `r` / Render instance seems like the only way to have a different HTMLTemplate / `application.html` which we need as this **countdown** page is a page without any header of footer and has a different stylesheet.

In my opinion the hand-crafted and simplistic HTML/CSS where easier to follow and adapt that the 60 000 lines and multiple plugins the bought template brang in but that might just be my point of view.

File: actions/app.go (hunk 1 above)
Comment (kiasaki) [reply actions/app.go:8]: Oops, yes I pull down my fork and `goimports` used the fork's path as the current directory of my editor was `$GOPATH/src/github.com/kiasaki/gcon`

**Comment by markbates** [comment]
For future reference if you you pass a second template name into `r.HTML` it will use that as a layout. 

`r.HTML("foo.html", "a-different-layout.html")`

**Pull request closed without merging**